src/frontend/js/modules/attachment/list/attachment-list.controller.js

```diff
@@ -1,12 +1,12 @@
-require('./attachment-list-providers.service.js');
```

Comment:
Why did you remove this? `esnAttachmentListProviders` is still there, right?

Reply:
It is already required in the app module.

Reply:
nvm i'm wrong